packages/client-core/i18n/en/user.json

```diff
@@ -192,6 +192,7 @@
       "lbl-settings": "Settings",
       "lbl-username": "Username",
       "usernameError": "Invalid username",
+      "usernameLengthError": "Username is too long",
```

Comment:
I suggest to be more explicit and to provide the user with feedback that indicates how to avoid the problem, example:

`Username must have XXX characters or less`

Reply:
good point! already addresed ty

![image](https://github.com/user-attachments/assets/e78f4968-c73e-45db-bd1f-09854b9016a0)